src/backend/Csrs.Api/Startup.cs

```diff
@@ -47,7 +47,7 @@ public static void ConfigureServices(this IServiceCollection services)
                 Title = "CSRS API",
                 Description = "BC Child Support Recalculation Service API",
             });
-
+            
```

Comment:
Why did we add this white space in this file. This commit shouldn't even  have made changes to `Startup.cs`

Reply:
White space was removed